packages/eslint-plugin-lit-a11y/lib/rules/tabindex-no-positive.js

```diff
@@ -47,16 +49,20 @@ const TabindexNoPositiveRule = {
 
           analyzer.traverse({
             enterElement(element) {
-              if (Object.keys(element.attribs).includes('tabindex')) {
-                const attributeValue = element.attribs.tabindex;
+              const attribs = Object.keys(element.attribs);
+              tabIndexAttributes.forEach(tabIndexAttribute => {
```

Comment:
Are all attributes in `element.attribs` lowercase already? Judging from the passing tests I guess they are, but is this something you checked? Just wanna make sure we dont run into issues with `.tabindex` vs `.tabIndex`

Reply:
yeah, keys in `element.attribs` are all lowercase. are we doing that?

Reply:
its probably parse5 doing it